core/api-changes.json

```diff
@@ -2,6 +2,24 @@
   "4.4.0": {
     "revapi": {
       "ignore": [
+        {
+          "code": "java.method.finalMethodAddedToNonFinalClass",
+          "new": "method java.lang.Object org.mule.runtime.core.api.streaming.bytes.factory.AbstractCursorStreamProviderFactory::of(org.mule.runtime.core.api.event.CoreEvent, java.io.InputStream, org.mule.runtime.api.component.location.ComponentLocation)",
+          "package": "org.mule.runtime.core.api.streaming.bytes.factory",
+          "classSimpleName": "AbstractCursorStreamProviderFactory",
+          "methodName": "of",
+          "elementKind": "method",
+          "justification": "MULE-18634: Add new method to receive ComponentLocator"
```

Comment:
this shouldn't be needed, because the class is `@NoExtend`. Seems to be an issue in the validator, can you create a jira for that? Also, explain that situation in this justification.